src/schematics/app/files/__directory__/__name__/BUILD.bazel__tmpl__

```diff
@@ -0,0 +1,35 @@
+package(default_visibility = ["//visibility:public"])
+load("@build_bazel_rules_angular//:defs.bzl", "ng_module")
+load("@build_bazel_rules_nrwl//:webpack.bzl", "webpack_bundle")
```

Comment:
you might want to have a re-export
(I'm thinking angular users would expect `index.bzl`)
so when you add another rule, they can add another parameter to their load statement, rather than one load statement per rule